contract/test/orca-contract.test.js

```diff
@@ -220,7 +171,7 @@ test('Start Orca contract', async t => {
   const { instance } = await E(zoe).startInstance(
     installation,
     {},
-    {},
+    { chainDetails },
```

Comment:
@turadg help?

```console
$ yarn lint
test/orca-contract.test.js:174:7 - error TS2322: Type 'Record<string, ChainInfo>' is not assignable to type 'never'.

174     { chainDetails },
```

Reply:
Mysterious. I checked this out and it looks like you're doing it right so maybe it's a bug in startInstance typedef.

For DOB purposes I'd just ts-expect-error

Reply:
- #66